app/people.js

```diff
@@ -61,6 +86,69 @@ const userAdded = {
   }
 }
 
+// Actions
+const inviteUser = {
+  key: "inviteUser",
+  noun: "User",
+  display: {
+    label: "Invite User",
+    description: "Invites user to the current portal."
+  },
+  operation: {
+    inputFields: [
+      {
+        label: "Email",
+        key: "email",
+        required: true
+      },
+      {
+        label: "Role",
+        key: "type",
+        required: true,
+        choices: {
+          "2": "User",
+          "4": "Power user",
+          "1": "Room admin",
+          "3": "DocSpace admin"
+        }
+      }
+    ],
+    /**
+     * @param {ZObject} z
+     * @param {Bundle<SessionAuthenticationData, InviteUserFields>} bundle
+     * @returns {Promise<Account>}
+     */
+    async perform(z, bundle) {
+      /**
+       * @param {Account[]} array
+       * @param {string} email
+       * @returns {Account|null}
+       */
+      const client = new Client(bundle.authData.baseUrl, z.request)
+      const people = new PeopleService(client)
+      /** @type {InviteUserBody} */
+      const body = {
+        invitations: [{
+          email: bundle.inputData.email,
+          type: bundle.inputData.type
+        }]
+      }
+      const accounts = await people.inviteUser(body)
+      let invitedUser
+      for (let i = 0; i < accounts.length; i++) {
+        if (accounts[i].displayName === bundle.inputData.email) {
+          invitedUser = accounts[i]
```

Comment:
When you find an invited user, you can break from the loop. Right?

Reply:
> When you find an invited user, you can break from the loop. Right?

Added break the loop